atlascope/core/models/investigation.py

```diff
@@ -12,6 +12,8 @@ class Investigation(TimeStampedModel, models.Model):
     description = models.TextField(max_length=5000, blank=True)
     datasets = models.ManyToManyField('Dataset', related_name='investigations')
     notes = models.TextField(max_length=5000, blank=True)
+    way_point = models.ForeignKey('Waypoint', on_delete=models.CASCADE, null=True)
+    tour = models.ForeignKey('Tour', on_delete=models.CASCADE, null=True)
```

Comment:
This confused me until I saw your through table setup below--see that comment for my thoughts on this change.

Reply:
resolved with 56d7e61af852626496a99eb9a5c4ec4b7b136f08